### PR TITLE
feat: Show number of active user sessions in the user manager plugin

### PR DESCRIPTION
**Pull request by connoratrug**
#### Checklist
- [x] Functionality works & meets specifications
- [x] Code reviewed
- [x] Code unit/integration/system tested
- [x] Migration step added in case of breaking change
- [x] User documentation updated
- [x] (If you have changed REST API interface) view-swagger.ftl updated
- [x] Test plan template updated
- [x] Clean commits
- [x] Added Feature/Fix to release notes

### CODE REVIEW

File: molgenis-security/src/main/java/org/molgenis/security/MolgenisWebAppSecurityConfig.java [hunk 1]
```diff
@@ -137,6 +140,10 @@ protected void configure(HttpSecurity http) throws Exception {
     requestCache.setRequestMatcher(createWebRequestMatcher());
     http.requestCache().requestCache(requestCache);
 
+    http.sessionManagement()
+        .maximumSessions(MAXIMUM_CONCURRENT_USER_SESSIONS)
```
Comment (fdlk) [inline molgenis-security/src/main/java/org/molgenis/security/MolgenisWebAppSecurityConfig.java:144]: That'll break concurrent use by functional / admin/admin accounts.

Comment (connoratrug) [reply molgenis-security/src/main/java/org/molgenis/security/MolgenisWebAppSecurityConfig.java:144]: was wondering about that , why would it break ?

Comment (connoratrug) [reply molgenis-security/src/main/java/org/molgenis/security/MolgenisWebAppSecurityConfig.java]: switched to using molgenis SecurityContextRegistry, removes need to set max sessions

File: molgenis-core-ui/src/main/java/org/molgenis/core/ui/admin/usermanager/UserManagerServiceImpl.java [hunk 2]
```diff
@@ -41,6 +44,12 @@ public void setActivationUser(String userId, Boolean active) {
     this.dataService.update(USER, user);
   }
 
+  @Override
+  @PreAuthorize("hasAnyRole('ROLE_SU','ROLE_MANAGER')")
```
Comment (fdlk) [inline molgenis-core-ui/src/main/java/org/molgenis/core/ui/admin/usermanager/UserManagerServiceImpl.java:48]: This check will only go off on a server where a non-su non-manager user is given plugin permissions on the user manager. In that rare case it'll cause a nasty 500 error breaking the complete screen.
I'd say simply remove the check.

**Review by fdlk — CHANGES_REQUESTED** [review]

File: molgenis-core-ui/src/main/java/org/molgenis/core/ui/admin/usermanager/UserManagerServiceImpl.java (hunk 2 above)
Comment (fdlk) [reply molgenis-core-ui/src/main/java/org/molgenis/core/ui/admin/usermanager/UserManagerServiceImpl.java:48]: Nevermind, the rest of the methods have the same annotation.

**Review by fdlk — APPROVED** [review]

**Pull request merged**